app/kuma-dp/cmd/testdata/bootstrap-config.golden.yaml

```diff
@@ -17,6 +17,11 @@ dynamicResources:
 node:
   cluster: backend
   id: dp-1.default.default
+  metadata:
+    dataplane.applications.metrics:
+    - name: app
+      port: 123
+      path: "/stats"
```

Comment:
we don't need this in metadata anymore, right?

Reply:
Removed